setup/environment36.yml

```diff
@@ -48,15 +48,15 @@ dependencies:
 - fastcache=1.0.2=py36_1
 - flask=0.12.2=py36_0
 - flask-cors=3.0.2=py36_0
-- freetype=2.5.5=2
+- freetype=2.5.5
 - get_terminal_size=1.0.0=py36_0
 - gevent=1.2.1=py36_0
 - greenlet=0.4.12=py36_0
 - h5py=2.7.0=np112py36_0
-- hdf5=1.8.17=1
+- hdf5
 - heapdict=1.0.0=py36_1
 - html5lib=0.999=py36_0
-- icu=54.1=0
+- icu
```

Comment:
Does `icu=58.2` work?
https://anaconda.org/anaconda/icu/files

Reply:
Actually, try removing this line completely - the packages that need it will automatically install the correct version.